### PR TITLE
SAK-33584 Signup add the meeting id and title to the default event tracking method

### PR DESCRIPTION
**Pull request by maurercw**
…g method

### CODE REVIEW

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/action/RemoveWaiter.java [hunk 1]
```diff
@@ -100,9 +100,8 @@ public SignupMeeting removeFromWaitingList(SignupMeeting meeting, SignupTimeslot
 			if (ToolManager.getCurrentPlacement() != null) {
 				String signupEventType = isOrganizer ? SignupEventTypes.EVENT_SIGNUP_REMOVE_ATTENDEE_WL_L
 						: SignupEventTypes.EVENT_SIGNUP_REMOVE_ATTENDEE_WL_S;
-				Utilities.postEventTracking(signupEventType, ToolManager.getCurrentPlacement().getContext()
-						+ " meetingId:" + meeting.getId() + " -removed from wlist on TS:"
-						+ SignupDateFormat.format_date_h_mm_a(timeslot.getStartTime()));
+				Utilities.postEventTracking(signupEventType, ToolManager.getCurrentPlacement().getContext(), meeting.getId(),
+						meeting.getTitle(), "-removed from wlist on TS:" + SignupDateFormat.format_date_h_mm_a(timeslot.getStartTime()));
```
Comment (jesusmmp) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/action/RemoveWaiter.java:104]: Hardcoded text?

```
-removed from wlist on TS:
```

File: signup/tool/src/java/org/sakaiproject/signup/tool/util/Utilities.java [hunk 2]
```diff
@@ -376,15 +376,16 @@ public static List<SelectItem> getMeetingTypeSelectItems(
 	 *            a detailed action info string
 	 */
 	public static void postEventTracking(String mainSignupEventType,
-			String eventActionInfo) {
+			String placementContext, Long meetingId, String meetingTitle, String eventActionInfo) {
 		if (postToDatabase) {
+			String eventContent = placementContext + " meetingId|title:" + meetingId + "|" + meetingTitle + " " + eventActionInfo;
```
Comment (jesusmmp) [inline signup/tool/src/java/org/sakaiproject/signup/tool/util/Utilities.java:381]: Hardcoded text?

```
meetingId|title:
```

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/action/RemoveWaiter.java (hunk 1 above)
Comment (maurercw) [reply signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/action/RemoveWaiter.java:104]: Hardcoded text was already present

File: signup/tool/src/java/org/sakaiproject/signup/tool/util/Utilities.java (hunk 2 above)
Comment (maurercw) [reply signup/tool/src/java/org/sakaiproject/signup/tool/util/Utilities.java:381]: This hardcoded text was essentially already present in other places, I just moved it here for consistency

**Review by adrianfish — APPROVED** [review]
This looks fine to me. Jesus is right about the hard coded text, but I think we should just open another ticket for that.

**Pull request merged**